mica/stats/update_acq_stats.py

```diff
@@ -477,8 +477,8 @@ def _get_obsids_to_update(check_missing=False):
             h5 = tables.open_file(table_file, "r")
             tbl = h5.root.data[:]
             h5.close()
-        except:
-            raise ValueError
+        except Exception as err:
+            raise ValueError from err
```

Comment:
Same question as above. Did you consider raise from None? Is there a reason for converting the exception to. ValueError without doing any other processing?

Reply:
Since this exception isn't handled at all, I think it makes more sense to just cut the try-except.

Reply:
the question is if there is client code that catches ValueError and not other types of exception.

Reply:
👍🏼